mkt/search/tests/test_views.py

```diff
@@ -1479,29 +1481,13 @@ def test_search_no_doc_type_passed(self):
 
     def test_search_homescreen(self):
```

Comment:
Add a homescreen to the test above, `test_search_no_doc_type_passed` to test that no homescreens are returned by default